tests/test_visitors/test_ast/test_operators/test_bitwise_boolean_mixup.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from wemake_python_styleguide.violations.best_practices import (
+    BitwiseAndBooleanMixupViolation,
+)
+from wemake_python_styleguide.visitors.ast.operators import BitwiseOpVisitor
+
+# add more cases for more usages of mixup
+
+
+@pytest.mark.parametrize('expression', [
+    'True | False',
+    '(x >= y) & True',
+    'first | (10 == second)',
+    '(11 != first) | (not False)',
+    '(1 or first) & (second and first)',
+    '(first > 0 & second != 0) | (other is None)',
+    '(5234 + 2345) & True',
+])
+def test_bitwise_boolean_mixup(
```

Comment:
Can we have some tests without `()`? Or will the operator order break?

Reply:
The operator order will break, `1 or first & second and first` will not raise a violation, as the left and right of the BitwiseOp are just Name variables, and I don't think there is any way to traverse the AST from Name variables. This means that there will be some false negatives, but I don't think that there's a better way.

Reply:
Ok 👍